### ISSUE TITLE
Jumping to location from within .opam/ doesn't work

### ISSUE THREAD

**Issue by loxs** [ocaml/merlin#849]
I've been following this guide: https://github.com/ocaml/merlin/wiki/Letting-merlin-locate-go-to-stuff-in-.opam

I have .cmx files for all of my opam packages and when I try to jump from the code I'm working on to a location which is in ~/.opam/, it works. But when I later try to jump further on from there, it doesn't work and instead it complains that there are no TAGS.

My setup is exactly this: https://github.com/ocaml/merlin/wiki/spacemacs-from-scratch

Is what I want possible, and/or is my configuration wrong?

Edit: using Opam 2 on Mac OS

**Comment by loxs** [comment]
After further investigation, this might not be a merlin problem (though merlin might be able to handle it better).

It seems to be more of a combination of opam and individual library issues.

For [Opium](https://github.com/rgrinberg/opium) it works as explained above.

But for [Tyxml](https://github.com/ocsigen/tyxml) it works, though when I first jump to it, Merlin takes me to the top modules of the project which make functors. If I then manually go to the directory `~/.opam/<switch name>/lib/tyxml/functor/` and I open some of the files there, I can readily jump between the files, jumping between them does work, although there is no `.merlin` file there.

I don't know if its appropriate to close this issue or there are some configurations I can make for improving this situation.

**Comment by trefis** [comment]
Sorry for the delay before answering.
The same issue was also discussed [here](https://github.com/ocaml/merlin/issues/863) and then on [discuss](https://discuss.ocaml.org/t/merlin-goto-definition-inside-installed-libraries/2788).

tl;dr: there's nothing to do on merlin's side. Jumping from inside the .opam seems weird to me.
